### PR TITLE
Adding Openshift 3.10 CI

### PR DESCRIPTION
**Pull request by gbenhaim**
And removing unneeded Openshift 3.7 file

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

**Comment by gbenhaim** [comment]
With https://github.com/kubevirt/kubevirt-ansible/pull/227 we can stop use the docker-storage role which requires the openshift-facts module. I hope that it will fix the issue.

**Comment by gbenhaim** [comment]
I'm blocked because of missing packages in Centos repos:
https://github.com/openshift/openshift-ansible/issues/8399

**Comment by rthallisey** [comment]
looks like an unset var in ci? ```Message:  last_checked_host: lago-master, last_checked_var: openshift_node_group_name;openshift_node_group_name must be defined for all nodes```

**Comment by nellyc** [comment]
This is a recent change which was done in openshift-ansible - 
all nodes must have openshift_node_group_name specified during both install and upgrade.

**Comment by cynepco3hahue** [comment]
Exactly
- master must have `openshift_node_group_name="node-config-master-infra"`
- node `openshift_node_group_name="node-config-compute"`
If you want that master will serve as compute node, you need to add all-in-one group name(do not remember exact name)

File: inventory [hunk 1]
```diff
@@ -43,6 +43,11 @@ ansible_service_broker_etcd_image_prefix=quay.io/coreos/
 ansible_service_broker_etcd_image_tag=latest
 # END ANSIBLE BROKER CONFIG
 
+[nodes:vars]
+# Mark all nodes as infra and compute nodes (oc >= 3.10)
+openshift_node_groups=[{'name': 'node-config-infra-compute', 'labels': ['node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true']}]
```
Comment (lukas-bednar) [inline inventory]: Based on error I see in jenkins, I would try to move this `openshift_node_groups` to `[OSEv3:vars]` section.

File: inventory [hunk 2]
```diff
@@ -24,6 +24,9 @@ openshift_master_admission_plugin_config={"ValidatingAdmissionWebhook":{"configu
 # Docker
 container_runtime_docker_storage_type=overlay2
 
+# OC 3.10 node and master labeling
+openshift_node_groups=[{'name': 'node-config-infra-compute', 'labels': ['node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true']}, {'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}]
```
Comment (cynepco3hahue) [inline inventory]: setting `openshift_node_groups` will set labels automatically according to the group, so you can remove it and you must have different groups, for `master`, `infra` and `compute` resources(in general every node can server as `infra` resource, but if we talk about non-HA deployment, the cluster must have only one master node)

**Comment by gbenhaim** [comment]
ci test please

File: stdci.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 substage:
   - openshift_3-9
```
Comment (nellyc) [inline stdci.yaml]: is there a point in keeping 3.9?

Comment (gbenhaim) [reply stdci.yaml]: Let's discuss it on today's meeting.
It can be removed in the following PR.

**Comment by gbenhaim** [comment]
Ready to be merged.

**Pull request merged**